pkg/daemons/agent/agent.go

```diff
@@ -172,10 +175,10 @@ func addFeatureGate(current, new string) string {
 	return current + "," + new
 }
 
-func checkCgroups() (root string, hasCFS bool, hasPIDs bool) {
+func checkCgroups() (kubeletRoot, runtimeRoot string, hasCFS, hasPIDs bool) {
```

Comment:
```suggestion
func checkCgroups() (kubeletRoot string, runtimeRoot string, hasCFS bool, hasPIDs bool) {
```

Reply:
This was changed here: https://github.com/k3s-io/k3s/pull/2668#discussion_r539604764